plugin/src/main/kotlin/net/octyl/ts2kt/gradle/tasks/DiscoverTs2ktExecutable.kt

```diff
@@ -100,10 +102,8 @@ open class DiscoverTs2ktExecutable : DefaultTask() {
             return ts2ktInstalled.canonicalPath
         }
 
-        findPath.find("npx")
-                ?: throw IllegalStateException("`npx` is not installed. Cannot run `ts2kt`.")
-        val packageOption = getPackageOption()
-        return "npx -p '$packageOption' ts2kt"
+        findPath.find("npx") ?: throw IllegalStateException("`npx` is not installed. Cannot run `ts2kt`.")
```

Comment:
Fairly sure that this needs the same treatment as `npm` -- `npx` is likely installed with an `exe` or `bat` extension, not available plain like this.

Reply:
Well, running inside a batch script, it runs as a subprocess of the existing cmd process. So it won't need to be specified. Unlike when you're originating the process initially.